package.json

```diff
@@ -1,33 +1,49 @@
 {
   "name": "shield-studies-addon-utils",
-  "description": "Utilities for building Shield-Study Mozilla Firefox addons.",
-  "version": "4.1.0",
-  "author": "Gregg Lind <glind@mozilla.com>",
+  "description": "Utilities for building Shield-Study Mozilla Firefox add-ons.",
+  "version": "5.0.0",
+  "author": "Mozilla",
+  "bin": {
+    "copyStudyUtils": "bin/copyStudyUtils.js"
+  },
   "bugs": {
     "url": "https://github.com/mozilla/shield-studies-addon-utils/issues"
   },
   "dependencies": {
-    "assert": "^1.4.1",
-    "fs-extra": "^4.0.0",
-    "fx-runner": "^1.0.7",
-    "geckodriver": "^1.8.0",
-    "mocha": "^3.4.2",
-    "path": "^0.12.7",
-    "selenium-webdriver": "^3.4.0"
+    "commander": "^2.15.1",
+    "fs-extra": "^6.0.1",
+    "npm": "^6.1.0",
```

Comment:
Typo? Utils will never depend on npm at runtime. Is npm 6 required as a global or dev dependency? If global: mention in readme. If dev: put as dev dependency.